GitSearch/Models/SpinnerViewController.swift

```diff
@@ -24,3 +35,28 @@ class SpinnerViewController: UIViewController {
             ])
     }
 }
+
+extension SpinnerViewController {
+    
+    func toggle() {
+
+        func add() {
+            owner.addChild(self)
+            self.view.frame = owner.view.frame
```

Comment:
There is an important distinction between [`frame` and `bounds`](https://developer.apple.com/library/archive/documentation/WindowsViews/Conceptual/ViewPG_iPhoneOS/WindowsandViews/WindowsandViews.html#//apple_ref/doc/uid/TP40009503-CH2-SW6)
```suggestion
            self.view.frame = owner.view.bounds
```